blueocean-dashboard/src/main/js/creation/ScmProvider.js

```diff
@@ -23,4 +23,11 @@ export default class ScmProvider {
         throw new Error('must implement getFlowManager');
     }
 
+    /**
+     * Called when the creation flow is about to end. Perform cleanup here.
+     */
+    destroyFlowManager() {
+        throw new Error('must implement destroyFlowManager');
+    }
+
```

Comment:
Is this called based on some user action that will never happen if the user just closes their tab, for example? We don't want to encourage implementers to put important stuff in here in that case.